src/cargo_command.rs

```diff
@@ -201,14 +201,14 @@ struct Target {
 }
 
 impl<'a> CargoBuildOutput<'a> {
-    pub(crate) fn artifact(&self) -> Result<PathBuf, Error> {
-        Ok(self.select_buildopt()?.artifact())
+    pub(crate) fn artifact(&self) -> Result<Vec<PathBuf>, Error> {
+        Ok(self.select_buildopt()?)
     }
 
     /// Selects the buildopt which fits with the requirements
     ///
     /// If there are multiple possible candidates, this will return an error
-    fn select_buildopt(&self) -> Result<&CargoBuildOutputElement, Error> {
+    fn select_buildopt(&self) -> Result<Vec<PathBuf>, Error> {
```

Comment:
Now that this function can return several paths the docs and names should be updated